src/test/java/sirius/web/http/TestRequest.java

```diff
@@ -228,6 +229,25 @@ public static TestRequest POST(String uri, JSONObject json) {
     }
 
     /**
+     * Creates a mock request simulating a POST on the given uri and form data while sending the given data.
+     *
+     * @param uri      the relative uri to call
+     * @param resource resource the data to send to the server
+     * @param formData the parameters be included in the post request
+     * @return an instance used to further specify the request to send
+     */
+    public static TestRequest POST(String uri, InputStream resource, Context formData) {
+        TestRequest result = POST(uri, formData);
+        try {
```

Comment:
duplicate code. irgendein grund das nicht in installContent zu schieben?

Reply:
außerdem - wie soll das gehen? man kann kein form UND content posten - dafür braucht man einen multipart form